latch/gql/execute.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import os
+from typing import Any, Dict, Optional
+
+import gql
+from gql.transport.requests import RequestsHTTPTransport
+
+# todo(ayush): make this understand sdk tokens too
```

Comment:
this todo should really be in vacuole - ideally we want to convert more stuff to gql so we need vac to understand sdk tokens since not all latch operations happen in an execution context